### PR TITLE
new package: arc-gtk-theme

### PR DESCRIPTION
**Pull request by kleo**
image

### CODE REVIEW

**Comment by TomJo2000** [comment]
We have a package request template for issues.
Unfortunately that doesn't automatically apply for PRs, 
so let me fill that out here really quick.

<h1><em></em></h1> 

### Why is it worth to add this package?

It's a GTK theme.

### Home page URL

https://github.com/jnsh/arc-theme/

### Source code URL

https://github.com/jnsh/arc-theme/

### Packaging policy acknowledgement

- [X] The project is actively developed.
- [X] The project has [existing packages](https://repology.org/projects) and is "well known".
- [X] Licensed under an [open source license](https://spdx.org/licenses/).
- [X] Not available through a language package manager: pip, npm, cpan, cargo, etc.
- [X] Not taking up too much disk space (< 100MiB per architecture, exceptions can be made)
- [X] Not duplicating the functionality of existing packages.
- [X] Not serving hacking, malware, phishing, spamming, spying, ddos functionality.
- [X] I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied if it is found lacking.

### Additional information

_No response_

File: x11-packages/arc-gtk-theme/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+TERMUX_PKG_VERSION=20221218
+TERMUX_PKG_SRCURL=https://github.com/jnsh/arc-theme/releases/download/${TERMUX_PKG_VERSION}/arc-theme-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=264570cc90a13b88d181334fb325f99a6ddc2a45392aa9ed7dba312aea042bb0
+TERMUX_PKG_DEPENDS="sassc"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_AUTO_UPDATE=false
+
+termux_step_configure(){
+    termux_setup_meson
```
Comment (TomJo2000) [inline x11-packages/arc-gtk-theme/build.sh]: This fails the package linter check.
Please use hard tabs for indentation for shell scripts,
as per the coding guidelines.
https://github.com/termux/termux-packages/wiki/Coding-guideline#script-formatting

**Review by TomJo2000 — DISMISSED** [review]

File: x11-packages/arc-gtk-theme/build.sh (hunk 1 above)
Comment (kleo) [reply x11-packages/arc-gtk-theme/build.sh]: already pushed fix. thanks!

**Comment by kleo** [comment]
thank you for providing the package request template @TomJo2000

**Comment by TomJo2000** [comment]
There is two other things I feel like still need to be brought up.

<h1><em></em></h1> 

- 1.) Do we even intend to package (more) toolkit themes in the main repository?

We currently package:
- `hicolor`, the FreeDesktop reference/fallback theme.
- `adwaita` the GTK 4 standard theme.
- `lxqt-themes` for LXQt
- `qt5-qtbase-gtk-platformtheme` the standard GTK 3-like theme for Qt5
- `papirus-icon-theme` I'm not sure for what reason other than popularity
- `fluent-gtk-theme` Also not sure why we package this.

It might be better to add further themes as packages in the [TUR](https://github.com/termux-user-repository/tur).

<h1><em></em></h1> 

- 2.) Please keep your commits squashed.

You can use either `git commit --amend` to add to your last commit.
Or if you need to edit multiple commits,
`git rebase -i HEAD~<n>` where `<n>` is the number of commits you wish to edit.
So for example: `git rebase -i HEAD~2` selects 2 commits starting from the `HEAD` (current) commit.

You can then select `s` or `squash` as the action for all commits below the one you want to combine into.

<details><summary>Example if needed.</summary>
<p>

image
image

</p>
</details>

After either `git commit --amend`, or `git rebase -i` you will need to *force push* the branch to send the changes upstream, since you have changed the commit history.
You can do so with `git push --force` or preferably `git push --force-with-lease=<your branchname>`
<sub><b>The second option will stop the push if there are changes upstream which haven't been fetched locally yet
This may happen if a maintainer rebases your PR on top of the latest</b> `master` <b>branch commit,
or otherwise makes fixes to your branch.</b></sub>

**Comment by kleo** [comment]
> 1.) Do we even intend to package (more) toolkit themes in the main repository?

Up to you guys. 

I just wanted to use this theme while on vnc rather than configure proot-distro with archlinux arm and arc-gtk-theme. 

I consider xfce, papirus-icon-theme and arc-gtk-theme a personal favorite.

File: x11-packages/arc-gtk-theme/build.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jnsh/arc-theme/
+TERMUX_PKG_DESCRIPTION="A flat theme with transparent elements"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=20221218
+TERMUX_PKG_SRCURL=https://github.com/jnsh/arc-theme/releases/download/${TERMUX_PKG_VERSION}/arc-theme-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=264570cc90a13b88d181334fb325f99a6ddc2a45392aa9ed7dba312aea042bb0
+TERMUX_PKG_DEPENDS="sassc"
```
Comment (TomJo2000) [inline x11-packages/arc-gtk-theme/build.sh:8]:
```suggestion
TERMUX_PKG_BUILD_DEPENDS="sassc"
```
`sassc` appears to be a make dependency only.
https://gitlab.archlinux.org/archlinux/packaging/packages/arc-gtk-theme/-/blob/main/PKGBUILD?ref_type=heads#L21

**Review by TomJo2000 — APPROVED** [review]

File: x11-packages/arc-gtk-theme/build.sh (hunk 2 above)
Comment (kleo) [reply x11-packages/arc-gtk-theme/build.sh]: fixed. thanks!

**Pull request merged**